.travis.yml

```diff
@@ -13,6 +13,7 @@ env:
 
 install:
   - npm install
+  - sudo pip install -r requirements.txt
```

Comment:
I would suggest you to instead set the language in travis as python and install node via `nvm install node`. That way both python and node would be able to run on travis properly.